client/src/App.js

```diff
@@ -24,7 +21,13 @@ function App() {
 
         <Route exact path='/my-lists'>
           <Lists />
+                  
         </Route>
+        <Route exact path='/contact-us'>
+          <Lists />
+                  
```

Comment:
customer just called an stated they want it.

Reply:
ok np